easybuild/framework/easyconfig/easyconfig.py

```diff
@@ -1226,6 +1226,31 @@ def get_parsed_multi_deps(self):
 
         return multi_deps
 
+    def find_dep_version_match(self, dep):
+        """Identify the correct version for this system from the choices provided. This returns the version to use."""
+        if isinstance(dep['version'], string_type):
```

Comment:
@edmondac I think it makes sense to only pass down `dep['version']` to this method, and then rename `dep` to `dep_version` in the method body...

Reply:
Agreed